src/Collections/WorkflowLogsCollection.php

```diff
@@ -0,0 +1,204 @@
+<?php
+
+namespace Pantheon\Terminus\Collections;
+
+use GuzzleHttp\Exception\GuzzleException;
+use Pantheon\Terminus\Exceptions\TerminusException;
+use Pantheon\Terminus\Models\Environment;
+use Pantheon\Terminus\Models\Site;
+use Pantheon\Terminus\Models\TerminusModel;
+use Pantheon\Terminus\Models\WorkflowLog;
+
+/**
+ * Class WorkflowLogsCollection
+ * @package Pantheon\Terminus\Collections
+ */
+class WorkflowLogsCollection extends SiteOwnedCollection implements \Iterator
+{
+    /**
+     *
+     */
+    public const PRETTY_NAME = 'Workflow Logs Collection';
+    /**
+     * @var string
+     */
+    protected $collected_class = WorkflowLog::class;
+    /**
+     * @var string
+     */
+    protected $url = 'sites/{site_id}/logs/workflows';
+    /**
+     * @var
+     */
+    private $current;
+
+    /**
+     * @throws TerminusException
+     */
+    public function __construct(Site $site)
+    {
+        parent::__construct(["site" => $site]);
+    }
+
+    /**
+     * @throws TerminusException
+     * @throws GuzzleException
+     */
+    public function fetch()
+    {
+        $result = $this->request()->request(
+            $this->getUrl(),
+            ['site_id' => $this->getSite()->id, "method" => "GET"]
+        );
+        $this->setData($result['data']);
+        return $this;
+    }
+
+    /**
+     * @param array $data
+     * @return void
+     */
+    public function setData(array $data = [])
+    {
+        $class = $this->getCollectedClass();
+        foreach ($data as $datum) {
+            $this->models[] = new $class($datum, ['collection' => $this]);
+        }
+    }
+
+
+    /**
+     * @throws TerminusException
+     * @throws GuzzleException
+     */
+    public function serialize()
+    {
+        return array_map(function ($model) {
+            return $model->serialize();
+        }, $this->models);
+    }
+
+    /**
+     * @return mixed|\Pantheon\Terminus\Models\TerminusModel
+     */
+    public function current(): ?TerminusModel
+    {
+        return $this->models[$this->current];
+    }
+
+    /**
+     * @return void
+     */
+    public function next(): void
+    {
+        $this->current++;
+    }
+
+    /**
+     * @return mixed|null
+     */
+    public function key(): int
+    {
+        return $this->current;
+    }
+
+    /**
+     * @return bool
+     */
+    public function valid(): bool
+    {
+        return isset($this->models[$this->current]);
+    }
+
+    /**
+     * @return void
+     */
+    public function rewind(): void
+    {
+        $this->current = 0;
+    }
+
+    public function findLatestFromOptionsArray(
+        $options = [
+            'type' => null,
+            'id' => null,
+            'commit_hash' => null,
+            'start' => 0,
+        ]
+    ): TerminusModel {
+        $start_time = $options['start'] ?? 0; // Default to 0 if not set
+        $wfl = $this->latest();
```

Comment:
We want to continually advance `$wfl` to the next item in the collection, as long as `$wfl->get('start_time') < $start_time;`. Unsure about the syntax for "next item" in Terminus collections, existing workflow:wait probably has similar applicable code, though.

Reply:
I see that this is correctly handled in `findLatestByProperty`; only the first `if` check is wrong.